infra/recipes/terraform/azure/modules/helm_release/main.tf

```diff
@@ -0,0 +1,55 @@
+resource "helm_release" "tables" {
+  chart = "tables"
+  name = "openhouse-tables-service"
+  repository = "../../../../k8s/helm"
+
+  values = ["${file("../../../../k8s/environments/azure/sandbox/tables/values.yaml")}"]
+
+  set {
+    name = "storageProperties.containerName"
+    value = var.container_name
+  }
+
+  set {
+    name = "storageProperties.storageAccountName"
+    value = var.storage_account_name
+  }
+
+  set {
+    name = "storageProperties.storageAccountKey"
+    value = var.storage_account_key
+  }
+
+  set_sensitive {
```

Comment:
So should I just change the database type for the tables service back to In Memory?

Reply:
it could be dummy, this configuration is specific to table service. But in the spirit of keeping cluster.yaml consistent across all services, taking back this comment.